Automattic/jetpack
# Instant Search: Add infinite scrolling

#### Changes proposed in this Pull Request:
This enables paging via a "Load More" button with infinite scrolling support.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Yes, this adds paging to Jetpack Instant Search results.

#### Testing instructions:
* Add `define( "JETPACK_SEARCH_PROTOTYPE", true );` to your wp-config.php.
* Ensure that your site has the Jetpack Pro plan and Jetpack Search enabled.
* Add a Jetpack Search widget to the Search page sidebar.
* Enter a query into the search widget.
* Ensure that the "Load more results" button renders like so: 
> <img width="549" alt="Screen Shot 2019-10-07 at 1 50 59 PM" src="https://user-images.githubusercontent.com/4044428/66343740-882ae200-e909-11e9-96a3-2357a313c872.png">
* Scroll to the bottom of the page. Ensure that more results are fetched and appended to the list of search results.
* Without scrolling to the bottom, click on the "Load more results" button. Ensure that more results are fetched and appended to the page.

#### Proposed changelog entry for your changes:
None.
